torch/_torch_docs.py

```diff
@@ -152,6 +152,29 @@ def merge_dicts(*dicts):
     tensor([ 1.2294,  2.2004,  1.3690,  1.7298])
 """.format(**common_args))
 
+add_docstr(torch.acosh,
+           r"""
+acosh(input, out=None) -> Tensor
+
+Returns a new tensor with the inverse hyperbolic cosine of the elements of :attr:`input`,
+All values in the input tensor must be greater than or equal to one.
```

Comment:
Maybe: "The domain of the inverse hyperbolic cosine is [1, inf), and values outside this range will be mapped to nan"?

Reply:
Since for `inf` we get `inf`, the docs now read:

```
Returns a new tensor with the inverse hyperbolic cosine of the elements of :attr:`input`.
The domain of the inverse hyperbolic cosine is `[1, inf)` and values outside this range
will be mapped to ``NaN``, except for `+ INF` for which the output is mapped to `+ INF`.
```

I think this can be made simpler, and I'm open to suggestions here.